CHANGELOG.md

```diff
@@ -14,8 +14,20 @@ You can find a list of previous releases on the [github releases](https://github
 ### Fixed
 - Fixed workflow replication for reset workflow (#5412)
 - Fixed visibility mode for admin when use Pinot visibility (#5441)
+- Fix timer-fixer, unfortunately broken in 1.2.5 (#5433)
 
 ## [1.2.5] - 2023-11-01
+
+### Caution:
+
+Prefer 1.2.4 or 1.2.6 if you have enabled the timer fixer.
```

Comment:
are you planning to publish 1.2.6 after merging this?

Reply:
Next will be v1.2.6-prerelease2. Usually we officially release after we test and roll out in our production clusters